keyboards/kprepublic/bm40hsrgb/keymaps/43keys/config.h

```diff
@@ -0,0 +1,72 @@
+// /*
+//   Set any config.h overrides for your specific keymap here.
+//   See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+// */
+#pragma once
+
+#define TAPPING_TERM 200 // time of holding key, milliseconds
+
+// Prevent normal rollover on alphas from accidentally triggering mods.
+#define IGNORE_MOD_TAP_INTERRUPT
+
+// Enable rapid switch from tap to hold, disables double tap hold auto-repeat.
+#define TAPPING_FORCE_HOLD
+
+// to reduce memory
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
```

Comment:
This isn't needed, as it's included by default, if console is off. 
```suggestion
```

Reply:
remove that code.